components/molecule/dropdownOption/src/index.js

```diff
@@ -118,7 +125,10 @@ MoleculeDropdownOption.propTypes = {
   onSelectKey: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
 
   /** Custom ref handler that will be assigned to the "target" element */
-  innerRef: PropTypes.object
+  innerRef: PropTypes.object,
+
+  /** Text with css clamp = 2 */
+  twoLinesText: PropTypes.bool
```

Comment:
Maybe naming could be improved by adding a prefix like these: `hasTwoLinesText ` or `withTwoLinesText`